components/enrichers/custom-annotation/main.go

```diff
@@ -0,0 +1,79 @@
+// Package main of the codeowners enricher
+// handles enrichment of individual issues with
+// the groups/usernames listed in the github repository
+// CODEOWNERS files.
+// Owners are matched against the "target" field of the issue
+package main
+
+import (
+	"flag"
+	"log"
+	"log/slog"
+	"strings"
+
+	apiv1 "github.com/ocurity/dracon/api/proto/v1"
+	"github.com/ocurity/dracon/components/enrichers"
+)
+
+const defaultAnnotation = ""
+
+var (
+	annotation      string
+	annotationValue string
+)
+
+func enrichIssue(i *apiv1.Issue) (*apiv1.EnrichedIssue, error) {
+	enrichedIssue := apiv1.EnrichedIssue{}
+	annotations := map[string]string{
+		annotation: annotationValue,
+	}
+
+	enrichedIssue = apiv1.EnrichedIssue{
+		RawIssue:    i,
+		Annotations: annotations,
+	}
+	return &enrichedIssue, nil
+}
+
+func run() error {
+	res, err := enrichers.LoadData()
+	if err != nil {
+		return err
+	}
+	if annotation == "" {
+		annotation = defaultAnnotation
+	}
+	for _, r := range res {
+		slog.Info("processing results for ", slog.Any("scan", r.ScanInfo))
+		enrichedIssues := []*apiv1.EnrichedIssue{}
```

Comment:
⛏️ since we know how many `res` we have we can init this more efficiently using `make([]*apiv1.EnrichedIssue, 0, len(res))`